src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -63,6 +67,7 @@ public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone
      */
     public JsonAdaptedPerson(Person source) {
         name = source.getName().fullName;
+        birthdate = source.getBirthdate().toString();
```

Comment:
To confirm, birth dates are stored in the same format as the input format (ie. dd-MM-YYYY) and this same stored string is passed into the constructor in line 115?

Reply:
Yes, that is correct.